<launch>
  <!-- 0-none, 1-states, 2-commands, 3-control_sp, 4-control_ff -->
  <param name="auxSend" value="3" type="int" />
  <arg name="joy_dev0" default="/dev/input/js0" />
  <arg name="joy_dev1" default="/dev/input/js1" />

  <group ns="joygroup0">
    <node pkg="joy" name="joy_node0" type="joy_node" >
      <param name="dev" value="$(arg joy_dev0)" />
    </node>
  </group>

  <group ns="joygroup1">
    <node pkg="joy" name="joy_node1" type="joy_node" >
      <param name="dev" value="$(arg joy_dev1)" />
    </node>
  </group>

  <node pkg="easyfly" name="joysticks2" type="joysticks2" output="screen"/>

</launch>
